noxfile.py

```diff
@@ -386,6 +387,43 @@ def autotyping(session: nox.Session, options: tuple[str, ...]) -> None:
     session.run("python", "-m", "autotyping", *options, *paths)
 
 
+@nox.session
+def monkeytype(session: nox.Session) -> None:
+    """
+    Automatically add type hints to a module as inferred from pytest.
+    Experimental.
+
+    Examples
+    --------
+    nox -s monkeytype -- plasmapy.particles.atomic
+    """
+
+    if not session.posargs:
+        session.error(
+            "Please add at least one module "
+            "`nox -s monkeytype -- plasmapy.particles.atomic`"
+        )
+
+    session.install(".[tests]")
+    session.install("MonkeyType", "pytest-monkeytype", "pre-commit")
+
+    database = pathlib.Path("./monkeytype.sqlite3")
+
+    if not database.exists():
+        session.log(f"File {database.absolute()} not found. Running MonkeyType.")
+        session.run(
+            "pytest", "-m", "not slow", f"--monkeytype-output={database.absolute()}"
```

Comment:
```suggestion
            "pytest", f"--monkeytype-output={database.absolute()}"
```
...for after this gets past the initial testing phase.